<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow
{
background-color: rgb(212, 185, 162);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">QStackedWidget::pane
{
background-color: rgb(212, 185, 162);
}
QStackedWidget
{
background-color: rgb(212, 185, 162);
}</string>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>172</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>-1</pointsize>
            <italic>false</italic>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 40px;
color: rgb(49, 17, 9);</string>
          </property>
          <property name="text">
           <string>Тестировщик</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>317</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="btn_page2">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 6px;
background-color: rgb(188, 139, 115);
color: rgb(49, 17, 9);
font-size: 24px;
}
QPushButton:hover{
background-color: rgb(156, 119, 98); 
}</string>
          </property>
          <property name="text">
           <string>Выбрать тест</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="btn_exit_2">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 6px;
background-color: rgb(188, 139, 115);
color: rgb(49, 17, 9);
font-size: 24px;
}
QPushButton:hover{
background-color: rgb(156, 119, 98); 
}</string>
          </property>
          <property name="text">
           <string>Выход</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="3">
         <widget class="QListWidget" name="listWidget">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QListWidget::pane
{
background-color: rgb(236, 205, 180);

}
QListWidget
{
background-color: rgb(236, 205, 180);
border: 2px solid rgb(49, 17, 9);
border-radius: 10px;
color: rgb(49, 17, 9);
}
QListWidget::item:hover{
background-color: rgb(156, 119, 98); 
}
</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btn_page3">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 6px;
background-color: rgb(188, 139, 115);
color: rgb(49, 17, 9);
font-size: 20px;
}
QPushButton:hover{
background-color: rgb(156, 119, 98); 
}</string>
          </property>
          <property name="text">
           <string>Выбрать</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="btn_deleteTest">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 6px;
background-color: rgb(188, 139, 115);
color: rgb(49, 17, 9);
font-size: 20px;
}
QPushButton:hover{
background-color: rgb(156, 119, 98); 
}</string>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btn_addTest">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 6px;
background-color: rgb(188, 139, 115);
color: rgb(49, 17, 9);
font-size: 20px;
}
QPushButton:hover{
background-color: rgb(156, 119, 98); 
}</string>
          </property>
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QPushButton" name="btn_page1">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 6px;
background-color: rgb(188, 139, 115);
color: rgb(49, 17, 9);
font-size: 20px;
}
QPushButton:hover{
background-color: rgb(156, 119, 98); 
}</string>
          </property>
          <property name="text">
           <string>Назад</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="7" column="2">
         <widget class="QPushButton" name="btn_exit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 6px;
background-color: rgb(188, 139, 115);
color: rgb(49, 17, 9);
font-size: 20px;
}
QPushButton:hover{
background-color: rgb(156, 119, 98); 
}</string>
          </property>
          <property name="text">
           <string>Выйти</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="btn_page1_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 6px;
background-color: rgb(188, 139, 115);
color: rgb(49, 17, 9);
font-size: 20px;
}
QPushButton:hover{
background-color: rgb(156, 119, 98); 
}</string>
          </property>
          <property name="text">
           <string>Главный экран</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <spacer name="verticalSpacer_10">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="btn_page6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 6px;
background-color: rgb(188, 139, 115);
color: rgb(49, 17, 9);
font-size: 20px;
}
QPushButton:hover{
background-color: rgb(156, 119, 98); 
}</string>
          </property>
          <property name="text">
           <string>Получить отчёт</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="2">
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QLabel" name="label_points">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 40px;
color: rgb(49, 17, 9);</string>
          </property>
          <property name="text">
           <string>Количество</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <spacer name="verticalSpacer_11">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="2">
         <spacer name="verticalSpacer_14">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QGridLayout" name="gridLayout_5"/>
      </widget>
      <widget class="QWidget" name="page_6">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="1" rowspan="2">
         <layout class="QFormLayout" name="formLayout_2"/>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer_15">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_16">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="5">
         <spacer name="horizontalSpacer_16">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="btn_report">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 6px;
background-color: rgb(188, 139, 115);
color: rgb(49, 17, 9);
font-size: 20px;
}
QPushButton:hover{
background-color: rgb(156, 119, 98); 
}</string>
          </property>
          <property name="text">
           <string>Создать отчёт</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="btn_page1_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 6px;
background-color: rgb(188, 139, 115);
color: rgb(49, 17, 9);
font-size: 20px;
}
QPushButton:hover{
background-color: rgb(156, 119, 98); 
}</string>
          </property>
          <property name="text">
           <string>Главный экран</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_7">
       <layout class="QGridLayout" name="gridLayout_7"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
